### PR TITLE
Security for all type "except entity" if has rol only hide menu

### PR DESCRIPTION
**Pull request by gonzakpo**
Security for type "route" only hide menu

### CODE REVIEW

File: src/Helper/MenuHelper.php [hunk 1]
```diff
@@ -45,6 +45,16 @@ protected function pruneAccessDeniedEntries(array $menuConfig, array $entitiesCo
                 unset($menuConfig[$key]);
                 continue;
             }
+            else if (
+                'route' === $entry['type']
+                && isset($entry['role'])
+                && !$this->adminAuthorizationChecker->isEasyAdminGranted(
```
Comment (alterphp) [inline src/Helper/MenuHelper.php]: Please use Symfony `security.authorization_checker` service. `AdminAuthorizationChecker` is intended to check access regarding an entity EasyAdmin config. You can not invent a list action role to deal with this case.

Comment (alterphp) [reply src/Helper/MenuHelper.php]: Adding a test to cover the use case would be really kindful too ;-)

Comment (gonzakpo) [reply src/Helper/MenuHelper.php]: Necessary? Now it's very simple and uses authorizationChecker:
```php
if (
    'route' === $entry['type']
    && isset($entry['role'])
    && !$this->authorizationChecker->isGranted($entry['role'])
)
```

Comment (gonzakpo) [reply src/Helper/MenuHelper.php]: Necessary? Now it's very simple and uses authorizationChecker:
```php
if (
    'route' === $entry['type']
    && isset($entry['role'])
    && !$this->authorizationChecker->isGranted($entry['role'])
)
```

**Pull request merged**